setup.cfg

```
@@ -10,5 +10,4 @@ url = https://github.com/tehmaze/ansi/
 
 [options]
 python_requires = >=3.6
-py_modules =
-    div_mod
+py_modules = ansi
```

Actually, `py_modules` is the key to use for a single `.py` file.

```suggestion
packages = ansi
```